Doc/library/http.server.rst

```diff
@@ -65,7 +65,7 @@ provides three different variants:
 
    The handler will parse the request and the headers, then call a method
    specific to the request type. The method name is constructed from the
-   request. For example, for the request method ``SPAM``, the :meth:`do_SPAM`
+   request. For example, for the request method ``SPAM``, the :meth:`!do_SPAM`
    method will be called with no arguments. All of the relevant information is
    stored in instance variables of the handler.  Subclasses should not need to
    override or extend the :meth:`__init__` method.
```

Comment:
Fix the other warning in this paragraph, while we're here? (We could link to the general docs on `__init__` methods in the datamodel, but I'm not sure they're really relevant here)

```suggestion
   override or extend the :meth:`!__init__` method.
```